src/adapters/TelnetAdapter.ts

```diff
@@ -580,6 +583,16 @@ export class TelnetAdapter {
                     value = util.removeTrailingNewline(value);
                     //the array/associative array print is a loop of every value, so handle that
                     children = this.getForLoopPrintedChildren(expression, value);
+                    if (highLevelType === HighLevelType.array) {
+                        children.unshift({
+                            name: '[[length]]',
```

Comment:
Should we call this `[[count]]` instead since that's the name of the method you would call when running this directly on a roku?

Reply:
Renamed to `[[count]]`